docs/getting_started/faq.md

```diff
@@ -0,0 +1,156 @@
+# Frequently Asked Questions
```

Comment:
Is this page necessary in the developer docu. We have https://docs.mapbender.org/develop/en/faq.html

Reply:
it's different content. Will be refactoed / unified in a future release.